src/sentry/api/endpoints/organization_discover.py

```diff
@@ -70,31 +98,18 @@ def post(self, request, organization, member):
 
         rollup = data.get('rollup')
```

Comment:
can you add the rest of these fields to the serializer? then you could probably just do something like `self.do_query(**serialized)`

Reply:
you can just make them `required=False`

Reply:
and you can set the default for `orderby` that way too: http://www.django-rest-framework.org/api-guide/fields/#default